webpack.config.js

```diff
@@ -126,7 +126,12 @@ const config = {
         // Allow the use of the real filename of the module being executed. By
         // default Webpack does not leak path-related information and provides a
         // value that is a mock (/index.js).
-        __filename: true
+        __filename: true,
+
+        // Emscripten generated glue code "rnnoise.js" expects node fs module,
+        // we need to specify this parameter so webpack knows how to properly
+        // interpret it when encountered.
+        fs: 'empty'
```

Comment:
Can you please pass it only on the rnnoise bundle?

Reply:
Fixed.